src/convertToObject.js

```diff
@@ -5,8 +5,24 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(stylesString) {
+  const lines = stylesString.trim().split('\n');
+  const cleanedLines = lines.filter((line) => line.trim() !== '');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy of the array before each operation. The filter method does not mutate the original array, so you can chain the methods directly without assigning it to 'cleanedLines'.